### PR TITLE
OSDOCS-11550: adds 4.14.37 relnotes Microshift

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.14

Issue:
[OSDOCS-11550](https://issues.redhat.com/browse/OSDOCS-11550)

Link to docs preview:
[4.1.4.37 async release notes](https://81931--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-14-release-notes.html#microshift-4-14-37-dp_release-notes)

QE review:
- N/A stock text, no other release notes

Additional information:
This PR also includes a backport of style manual date formatting. See https://www.ibm.com/docs/en/ibm-style?topic=measurement-dates-times#general-guidance-for-dates.

### CODE REVIEW

File: microshift_release_notes/microshift-4-14-release-notes.adoc [hunk 1]
```diff
@@ -611,4 +611,13 @@ Issued: 2024-09-12
 
 {product-title} release 4.14.36 is now available. The list of enhancements and bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:6410[RHBA-2024:6410] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:6406[RHSA-2024:6406] advisory.
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[id="microshift-4-14-37-dp_{context}"]
+=== RHBA-2024:6690 - {microshift-short} 4.14.37 bug fix and security update advisory
+
+Issued: 2024-09-19
```
Comment (JoeAldinger) [inline microshift_release_notes/microshift-4-14-release-notes.adoc:619]: I know this is consistent with the MS template for RNs but I wanted to point out that OCP core has switched to the following [guidelines](https://docs.openshift.com/container-platform/4.16/release_notes/ocp-4-16-release-notes.html#ocp-4-16-11_release-notes:~:text=Issued%3A-,11%20September%202024,-OpenShift%20Container%20Platform) for dates (ISG).

**Review by JoeAldinger — COMMENTED** [review]
/lgtm, one question

File: microshift_release_notes/microshift-4-14-release-notes.adoc (hunk 1 above)
Comment (aravipra) [reply microshift_release_notes/microshift-4-14-release-notes.adoc]: Done. Thank you for pointing out.

**Pull request merged**